updater/lib/dependabot/service.rb

```diff
@@ -1,5 +1,6 @@
 # frozen_string_literal: true
 
+require "raven"
```

Comment:
🔎 Is this a missed import from a previous PR?

Reply:
Yeah, it should have moved in a previous PR. This file is always included after `require "raven"` in the [binstub](https://github.com/dependabot/dependabot-core/blob/main/updater/bin/update_files.rb#L7) so it didn't cause any issues.